### PR TITLE
OD-572 [Feature] Added translation to Widget-LFD

### PR DESCRIPTION
**Pull request by OleksandrLavrynenko**
@sofiiakvasnevska @inna-bieshulia

OD-572 https://weboo.atlassian.net/browse/OD-572

## Description
Added user translations in LFD components

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@upplabs-alex-levchenko

### CODE REVIEW

**Review by tonytlwu — CHANGES_REQUESTED** [review]

**Review by tonytlwu — APPROVED** [review]
@sofiiakvasnevska @inna-bieshulia Translation looks ready. This can be tested once API is ready.

File: templates/build/news-feed-base.build.hbs [hunk 1]
```diff
@@ -41,7 +41,7 @@
     <div class="search-holder">
       <i class="fa fa-search"></i>
       <input type="search" name="search-feed" class="form-control search-feed focus-outline" tabindex="0" placeholder="Search..." required >
```
Comment (skrupnyk) [inline templates/build/news-feed-base.build.hbs:43]: placeholder="{{T "widgets.list.dynamic.search.placeholder"}}"

File: templates/build/simple-list-base.build.hbs [hunk 2]
```diff
@@ -41,7 +41,7 @@
     <div class="search-holder">
       <i class="fa fa-search"></i>
       <input type="search" name="search-feed" class="form-control search-feed focus-outline" tabindex="0" placeholder="Search..." required >
```
Comment (skrupnyk) [inline templates/build/simple-list-base.build.hbs:43]: placeholder="{{T "widgets.list.dynamic.search.placeholder"}}"

File: templates/build/small-card-base.build.hbs [hunk 3]
```diff
@@ -46,7 +46,7 @@
     <div class="search-holder">
       <i class="fa fa-search"></i>
       <input type="search" name="search-feed" class="form-control search-feed focus-outline" tabindex="0" placeholder="Search..." required >
```
Comment (skrupnyk) [inline templates/build/small-card-base.build.hbs:48]: placeholder="{{T "widgets.list.dynamic.search.placeholder"}}"

**Review by skrupnyk — COMMENTED** [review]
https://github.com/Fliplet/fliplet-widget-dynamic-lists/blob/60ff1610f7476ae9751221cfa621f0bd826c19a6/js/layout-javascript/news-feed-code.js#L2669

T("widgets.list.dynamic.comments.title")

**Review by skrupnyk — COMMENTED** [review]
https://github.com/Fliplet/fliplet-widget-dynamic-lists/blob/60ff1610f7476ae9751221cfa621f0bd826c19a6/js/layout-javascript/simple-list-code.js#L2616

T("widgets.list.dynamic.comments.title")

File: translation.json [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+          "survey": "Survey",
+          "questions": "Questions",
+          "linkText": "Tap to open",
+          "uploaded": "Uploaded",
+          "edit": "Edit",
+          "delete": "Delete"
+        },
+        "comments": {
+          "listTitle": "Comments",
+          "offline": "You are offline",
```
Comment (skrupnyk) [inline translation.json:32]: Add `"placeholder": "Type comment",`

File: templates/build/news-feed-base.build.hbs [hunk 5]
```diff
@@ -186,9 +197,9 @@
       <textarea class="form-control" rows="1" placeholder="Type comment" data-comment-body></textarea>
```
Comment (skrupnyk) [inline templates/build/news-feed-base.build.hbs:197]: `placeholder="{{T "widgets.list.dynamic.comments.placeholder"}}"`

File: templates/build/simple-list-base.build.hbs [hunk 6]
```diff
@@ -186,9 +196,9 @@
       <textarea class="form-control" rows="1" placeholder="Type comment" data-comment-body></textarea>
```
Comment (skrupnyk) [inline templates/build/simple-list-base.build.hbs:196]: `placeholder="{{T "widgets.list.dynamic.comments.placeholder"}}"`

**Review by skrupnyk — COMMENTED** [review]
https://github.com/Fliplet/fliplet-widget-dynamic-lists/blob/60ff1610f7476ae9751221cfa621f0bd826c19a6/js/layout-javascript/news-feed-code.js#L39

`T("widgets.list.dynamic.loading")`

**Review by skrupnyk — COMMENTED** [review]
https://github.com/Fliplet/fliplet-widget-dynamic-lists/blob/60ff1610f7476ae9751221cfa621f0bd826c19a6/js/layout-javascript/simple-list-code.js#L31

`T("widgets.list.dynamic.loading")`

**Review by tonytlwu — APPROVED** [review]

**Pull request merged**